apps/meteor/app/federation-v2/server/infrastructure/rocket-chat/hooks/index.ts

```diff
@@ -160,7 +160,7 @@ export class FederationHooks {
 				) {
 					return message;
 				}
-				if (!isEditedMessage(message)) {
+				if (!isEditedMessage(message) || !message.editedBy) {
```

Comment:
any reason to not add the check `!message.editedBy` into `isEditedMessage` function?

Reply:
Yes, three failing Federation unit tests when the type guard is more stricter.